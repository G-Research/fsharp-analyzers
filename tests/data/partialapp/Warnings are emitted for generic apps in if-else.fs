module M

let myFunc x y = x + y
let myGenFunc<'t> x y z = x + y + z

let _ = (if true then myGenFunc<int> 23 42 else myFunc 23) 88 // should warn
