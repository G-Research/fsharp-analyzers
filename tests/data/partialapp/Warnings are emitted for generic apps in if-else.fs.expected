GRA-PARTAPP-001 | Warning | (6,22 - 6,31) | Partial application should not be used.
GRA-PARTAPP-001 | Warning | (6,48 - 6,54) | Partial application should not be used.
